src/components/explain-body/explain-body.less

```diff
@@ -1,5 +1,13 @@
 @import (reference) "~less/compass/_theme.less";
 
 .explain-body {
-  height: 100%;
+  padding: 0 15px;
+  position: relative;
+  flex-grow: 1;
+  flex-shrink: 1;
+  flex-basis: auto;
+  flex-direction: column;
+  display: flex;
+  overflow: auto;
+  background: #f5f6f7;
```

Comment:
I am not 100% sure if removing `height: 100%` is what's causing this, but I do get the background of the thing going all the way to the bottom of my screen. e.g. the explain part is the grey, and the rest is white:
<img width="510" alt="Screen Shot 2019-10-30 at 7 08 37 PM" src="https://user-images.githubusercontent.com/8107784/67886038-fe0d1c80-fb48-11e9-8f6f-d5a3fd6bccbf.png">

Reply:
This problem is no longer there when the query is a lot more complicated and needs to grow further, but on shorter explain plans it only comes half way.